cli/tests/error_worker_permissions_local.ts.out

```diff
@@ -0,0 +1,2 @@
+[WILDCARD]
+error: Uncaught (in worker "") read access to "[WILDCARD]worker_types.ts", run again with the --allow-read flag
```

Comment:
This error message really isn't great. It don't actually specifically state that the permission was denied. Something we should address in a follow up PR.

Reply:
It should say `error: Uncaught (in worker "") PermissionDenied: read access to ...`. Not sure if this PR did something to omit it or it's an earlier regression.

Reply:
It isn't this PR... I am not touching the error:

```rust
    if target_lib == TargetLib::Worker {
      runtime_permissions.check_specifier(&specifier)?;
    }
```

So very appropriate to fix in a follow up PR.

Reply:
I've kind of fixed this locally, but I think it's a wontfix. There are error `Results` throughout cli which are passed to JS in some cases and terminated upon in Rust in other cases (like this one), and there isn't really a good way of selectively unifying the output that always works the way we want it to. The issue has been present for a while, it's mostly fine as it is.